test/regress/regress0/quantifiers/Makefile.am

```diff
@@ -95,13 +95,15 @@ TESTS =	\
 	qbv-test-invert-bvor.smt2 \
 	qbv-test-invert-bvlshr-0.smt2 \
  	qbv-test-invert-bvurem-1.smt2 \
-	qbv-simple-2vars-vo.smt2 \
 	qbv-test-invert-concat-0.smt2 \
 	qbv-test-invert-concat-1.smt2 \
 	qbv-test-invert-shl.smt2 \
 	qbv-test-invert-udiv-0.smt2 \
 	qbv-test-invert-udiv-1.smt2 \
-	qbv-test-urem-rewrite.smt2
+  qbv-simple-2vars-vo.smt2 \
```

Comment:
Seems that while resolving conflicts I introduced a misalignment here. Sorry about that.

Reply:
No worries